### PR TITLE
chore(condo): DOMA-5783 models improvements

### PR DESCRIPTION
**Pull request by AleX83Xpert**
new relation between NewsItem and NewsItemScope
improved access checking

### CODE REVIEW

File: apps/condo/domains/news/schema/NewsItem.js [hunk 1]
```diff
@@ -65,6 +65,13 @@ const NewsItem = new GQLListSchema('NewsItem', {
             type: 'DateTimeUtc',
         },
 
+        scopes: {
```
Comment (akarjakin) [inline apps/condo/domains/news/schema/NewsItem.js:68]: Is migration missing here ?

Comment (AleX83Xpert) [reply apps/condo/domains/news/schema/NewsItem.js:68]: No migration needed. Created fields are enough.

File: apps/condo/domains/news/schema/NewsItemScope.js [hunk 2]
```diff
@@ -16,16 +14,16 @@ const NewsItemScope = new GQLListSchema('NewsItemScope', {
 
         newsItem: {
             schemaDoc: 'The news item to control access for',
-            type: Relationship,
-            ref: 'NewsItem',
+            type: 'Relationship',
```
Comment (pahaz) [inline apps/condo/domains/news/schema/NewsItemScope.js:17]: 🥇

File: apps/condo/domains/news/schema/NewsItemScope.js [hunk 3]
```diff
@@ -16,16 +14,16 @@ const NewsItemScope = new GQLListSchema('NewsItemScope', {
...
+            ref: 'NewsItem.scopes',
             isRequired: true,
             knexOptions: { isNotNullable: true }, // Required relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
         },
 
         property: {
             schemaDoc: 'Filter on Resident by property, who can read news',
-            type: Relationship,
+            type: 'Relationship',
```
Comment (pahaz) [inline apps/condo/domains/news/schema/NewsItemScope.js:26]: its ks5/ks6 compatibility ;)

**Review by pahaz — APPROVED** [review]

**Pull request merged**